eZ/Publish/Core/REST/Client/Values/User/User.php

```diff
@@ -35,6 +35,14 @@ public function getVersionInfo()
     }
 
     /**
+     * {@inheritdoc}
```

Comment:
@bdunogier regarding `{@inheritDoc}`, in french: https://www.matthecat.com/inheritdoc-vs-inheritdoc/